Gemfile

```diff
@@ -2,6 +2,5 @@ source "https://rubygems.org"
 
 gemspec
 
-gem 'rails', github: 'rails/rails'
-gem 'arel',  github: 'rails/arel'
+gem 'activemodel', '>= 5.x'
 gem 'mocha'
```

Comment:
True. I generally keep dev dependencies in the Gemfiles of my own projects. Not sure what you would prefer for this.